### DISCUSSION TITLE
What is `has_se`?

### DISCUSSION THREAD

**Discussion by ClayShoaf** [tinygrad/tinygrad#3777] [General]
I am trying to train efficientnet on my own data and, while combing through the example code, I came across this boolean argument `has_se`. I can't find anything in the docs or [anywhere else in this repo](https://github.com/search?q=repo%3Atinygrad%2Ftinygrad%20has_se&type=code) that isn't related to efficientnet.

So, what is it and, if I'm trying to train on my own dataset, should I set it to true or false?

**Comment by wozeparrot** [comment]
It's whether the block has a squeeze-excite block in it: https://paperswithcode.com/method/squeeze-and-excitation-block

**Reply by ClayShoaf** [reply]
Thank you for the reply. This is a little deep in the weeds for my little test project. I know how to set up training data, but actually training it and using the model that comes out of the other end has been a real challenge. Unfortunately, everything that isn't generative AI isn't as flashy, so there aren't a million tutorials on YouTube for this stuff.
